.gitmodules

```diff
@@ -1,23 +1,23 @@
 [submodule "salt_stack/salt/boss-tools/files/boss-tools.git"]
 	path = salt_stack/salt/boss-tools/files/boss-tools.git
 	url = ../boss-tools.git
-	branch = master	
+	branch = ingest-lambda-timeout	
```

Comment:
I recently learned that we don't need to update .gitmodules if we checkout the submodule commit directly. This is only necessary for using the convenience command: `git submodule update --remote`. We may want to consider revising the integration rebuild instructions since it seems like we're often not working off of HEAD nowadays.

Reply:
Ok.  But some other way to know there is other versions out there wouldn't we?

Reply:
At deployment time, we probably want to poll all the devs and find out which commit, in each repo, is ready for prime time.

Reply:
It is also used when you clone the repository and do the initial checkout. I don't think we need to poll anyone, as we should be working on the HEAD of the master branch.